index.bs

```diff
@@ -0,0 +1,221 @@
+<pre class=metadata>
+Title: Layout Instability API
+Status: ED
+Shortname: layout-instability
+Group: WICG
+Level: 1
+Editor: Nicolás Peña Moreno, Google, https://google.com, npm@chromium.org
```

Comment:
```suggestion
Editor: Nicolás Peña Moreno, Google https://google.com, npm@chromium.org
```
ditto the ones below, otherwise the names link to Google and the emails show as plaintext.

Reply:
Done.